developer-docs-site/docs/guides/handle-aptos-errors.md

```diff
@@ -0,0 +1,166 @@
+|ECOIN_INFO_ALREADY_PUBLISHED|`CoinType` is already initialized as a coin.|?|
+|ECOIN_INFO_NOT_PUBLISHED|`CoinType` hasn't been initialized as a coin.|?|
+|ECOIN_STORE_ALREADY_PUBLISHED|Account already has `CoinStore` registered for `CoinType`.|?|
+|ECOIN_STORE_NOT_PUBLISHED|Account hasn't registered `CoinStore` for `CoinType`.|?|
+|EINSUFFICIENT_BALANCE|Not enough coins to complete transaction.|?|
+|EDESTRUCTION_OF_NONZERO_TOKEN|Cannot destroy non-zero coins.|?|
+|EZERO_COIN_AMOUNT|Coin amount cannot be zero.|?|
+|EFROZEN|CoinStore is frozen. Coins cannot be deposited or withdrawn.|?|
+|ECOIN_SUPPLY_UPGRADE_NOT_SUPPORTED|Cannot upgrade the total supply of coins to different implementation.|?|
+|ECOIN_NAME_TOO_LONG|Name of the coin is too long.|?|
+|ECOIN_SYMBOL_TOO_LONG|Symbol of the coin is too long.|?|
+
+## Aptos tokens
+
+|Error |Message  | Action|
+--- | :---: | :---:|
+|EALREADY_HAS_BALANCE|The token has balance and cannot be initialized.|?|
+|ECOLLECTIONS_NOT_PUBLISHED|There isn't any collection under this account.|?|
+|ECOLLECTION_NOT_PUBLISHED|Cannot find collection in creator's account.|?|
+|ECOLLECTION_ALREADY_EXISTS|The collection already exists.|?|
+|ECREATE_WOULD_EXCEED_COLLECTION_MAXIMUM|Exceeds the collection's maximal number of token_data.|?|
+|EINSUFFICIENT_BALANCE|Insufficient token balance.|?|
+|EINVALID_TOKEN_MERGE|Cannot merge the two tokens with different token IDs.|?|
+|EMINT_WOULD_EXCEED_TOKEN_MAXIMUM|xceed the token data maximal allowed.|?|
+|ENO_BURN_CAPABILITY|No burn capability.|?|
+|ETOKEN_DATA_ALREADY_EXISTS|TokenData already exists.|?|
+|ETOKEN_DATA_NOT_PUBLISHED|TokenData not published.|?|
+|ETOKEN_STORE_NOT_PUBLISHED|TokenStore doesn't exist.|?|
+|ETOKEN_SPLIT_AMOUNT_LARGER_THAN_TOKEN_AMOUNT|Cannot split token to an amount larger than its amount.|?|
+|EFIELD_NOT_MUTABLE|The field is not mutable.|?|
+|ENO_MUTATE_CAPABILITY|Not authorized to mutate.|?|
+|ENO_TOKEN_IN_TOKEN_STORE|Token not in the token store.|?|
+|EUSER_NOT_OPT_IN_DIRECT_TRANSFER|User didn't opt-in direct transfer.|?|
+|EWITHDRAW_ZERO|Cannot withdraw 0 token.|?|
+|ENFT_NOT_SPLITABLE|Cannot split a token that only has 1 amount.|?|
+|ENO_MINT_CAPABILITY|No mint capability|?|
+|ECOLLECTION_NAME_TOO_LONG|The collection name is too long.|?|
+|ENFT_NAME_TOO_LONG|The NFT name is too long.|?|
+|EURI_TOO_LONG|The URI is too long.|?|
+|ENO_DEPOSIT_TOKEN_WITH_ZERO_AMOUNT|Cannot deposit a token with 0 amount.|?|
+|ENO_BURN_TOKEN_WITH_ZERO_AMOUNT|Cannot burn 0 token.|?|
+|EWITHDRAW_PROOF_EXPIRES|Withdraw proof expires.|?|
+|EOWNER_CANNOT_BURN_TOKEN|Token is not burnable by owner.|?|
+|ECREATOR_CANNOT_BURN_TOKEN|Token is not burnable by creator.|?|
+|ECANNOT_UPDATE_RESERVED_PROPERTY|Reserved fields for token contract. Cannot be updated by user.|?|
+|EURI_TOO_SHORT|URI too short.|?|
+|ETOKEN_OFFER_NOT_EXIST|Token offer doesn't exist.|?|
```

Comment:
A few points:
1. Does the developer see only the "Error" column or do they also see the "Message" column contents also, when they run into error?
2. If they only see the Error column, then the Message column is actually saying, "that means this", right? If so, then my suggestion is to change the header "Message" to "Meaning". 
3. Over time we will fill up the "Action" column, right? Could this be stated as "Suggested Action" or "Recommended Action"?

Reply:
Thanks, Raj.

> A few points:
> 
> 1. Does the developer see only the "Error" column or do they also see the "Message" column contents also, when they run into error?

I believe they see both the error and message.

> 2. If they only see the Error column, then the Message column is actually saying, "that means this", right? If so, then my suggestion is to change the header "Message" to "Meaning".

I am fine with this. Greg asked for this catalog. Let's see what he says.

> 3. Over time we will fill up the "Action" column, right? Could this be stated as "Suggested Action" or "Recommended Action"?

Yes, Greg has offered to help populate the Action column. Perhaps we call it "Resolution" instead?

In other news:

A. We should likely remove/replace the Diem ref.

B. I left out those with no description message as not much value in including them.

C. I added the contents verbatim except obvious typos ex. addres

D. I can edit the text next. Was hoping to get Greg's input.
Ex. u64 > 'unsigned 64 bits' value

E. Some contents remain unclear to me:
“Cannot split a token that only has 1 amount”
Should that be “that has a value of 1”

F. We can move the source file links to the respective tables, between them and their headers if that would make more sense.

Reply:
E. I think the item "E" will be clear if we state the units for "1", but Greg will know. (if nft, is this saying a collection of 1 item, or is it 1 unit of token?)
F. For now as it is now feels okay. Over time we might think of it? Either way, low care factor in my view, for now. 
Thanks!

Reply:
I went with Meaning and Recommendation. WDYT?